cave/reader/smac2_reader.py

```diff
@@ -0,0 +1,195 @@
+import re
+import os
+import shutil
+import csv
+import numpy as np
+import pandas as pd
+
+from ConfigSpace import Configuration, c_util
+from ConfigSpace.hyperparameters import IntegerHyperparameter, FloatHyperparameter
+from smac.optimizer.objective import average_cost
+from smac.utils.io.input_reader import InputReader
+from smac.runhistory.runhistory import RunKey, RunValue, RunHistory, DataOrigin
+from smac.utils.io.traj_logging import TrajLogger
+from smac.scenario.scenario import Scenario
+
+from cave.reader.base_reader import BaseReader, changedir
+from cave.utils.csv2rh import CSV2RH
+
+class SMAC2Reader(BaseReader):
+
+    def get_scenario(self):
+        run_1_existed = os.path.exists('run_1')
+        in_reader = InputReader()
+        # Create Scenario (disable output_dir to avoid cluttering)
+        scen_fn = os.path.join(self.folder, 'scenario.txt')
+        scen_dict = in_reader.read_scenario_file(scen_fn)
+        scen_dict['output_dir'] = ""
+        with changedir(self.ta_exec_dir):
+            self.logger.debug("Creating scenario from \"%s\"", self.ta_exec_dir)
+            scen = Scenario(scen_dict)
+
+        if (not run_1_existed) and os.path.exists('run_1'):
+            shutil.rmtree('run_1')
+        self.scen = scen
+        return scen
+
+    def get_runhistory(self, cs):
+        """
+        Returns:
+        --------
+        (rh, validated_rh): RunHistory, Union[False, RunHistory]
+            runhistory and (if available) validated runhistory
+        """
+
+        validated_rh = False
```

Comment:
This stays always false. We should also parse Validated SMAC2 output.

Reply:
I don't know about the format. If you have any validated smac2-data, I'd be glad to work on it.

Reply:
I'll search for some on Monday.